dist/stat_data.py

```diff
@@ -976,6 +976,8 @@ def __init__(self, name, desc, flags=''):
     # Cache and eviction statistics
     ##########################################
     CacheStat('cache_bytes_dirty', 'tracked dirty bytes in the cache', 'no_clear,no_scale,size'),
```

Comment:
The `cache_bytes_dirty` is formed internally by the WiredTiger using the `cache_bytes_dirty_leaf` and `cache_bytes_dirty_internal`. As we are publishing these two individual values, is it possible to construct this value in the T2 itself by removing this `cache_bytes_dirty` statistic? I am fine to leave it if it is not possible in T2.

Reply:
Yes, this is possible. It also shouldn't cause version dependencies because both this old stat and the new derived stat can be defined and each one will only display if the data exists for it.

Reply:
In that case, I could also remove `cache_bytes_dirty_total`. 

The comment in stat_data.py says: "bytes dirty in the cache cumulative"

However, it's actually just the same as `cache_bytes_dirty` just summed at different locations.

Reply:
On second thoughts, the data source stat must be computed within WiredTiger we shouldn't remove the total sum.

Reply:
As discussed: This stat can still manually be computed from the data source stats if necessary. The motivation behind removing this is to reduce the noise of having excessive and repetitive statistics.